components/UI/Buttons/ButtonLink/index.js

```diff
@@ -10,16 +12,24 @@ const ButtonLink = React.forwardRef(({
   prefetch,
   target,
   onClick,
-}, ref) => (
-  <Link
+}, ref) => {
+  const { trackEvent } = useAnalytics()
+
+
```

Comment:
lets make sure we keep our spacing consistent, here 2 spaces arent needed, consider only having one.